src/index.ts

```diff
@@ -78,6 +78,7 @@ export { default as SearchBar } from './components/SearchBar';
 export { default as SearchButton } from './components/SearchButton';
 export { default as SearchField } from './components/SearchField';
 export { default as Section } from './components/Section';
+export { default as Select } from './components/Select';
```

Comment:
Nice lol

Reply:
Btw our plop command will generate recommended files and insert this line for you i.e. `yarn plop Loader` or `yarn plop Select`

Reply:
lolz, yeah. For that select component I made a copy of the Dropdown directory, so bypassed plop in that case